x/wasm/internal/keeper/test_common.go

```diff
@@ -340,28 +340,36 @@ func RandomBech32AccountAddress(_ *testing.T) string {
 	return addr.String()
 }
 
-type HackatomExampleContract struct {
+type ExampleContract struct {
 	InitialAmount sdk.Coins
 	Creator       crypto.PrivKey
 	CreatorAddr   sdk.AccAddress
 	CodeID        uint64
 }
 
-func StoreHackatomExampleContract(t *testing.T, ctx sdk.Context, keepers TestKeepers) HackatomExampleContract {
+func StoreHackatomExampleContract(t *testing.T, ctx sdk.Context, keepers TestKeepers) ExampleContract {
+	return StoreExampleContract(t, ctx, keepers, "./testdata/hackatom.wasm")
+}
+
+func StoreBurnerExampleContract(t *testing.T, ctx sdk.Context, keepers TestKeepers) ExampleContract {
+	return StoreExampleContract(t, ctx, keepers, "./testdata/burner.wasm")
+}
+
+func StoreExampleContract(t *testing.T, ctx sdk.Context, keepers TestKeepers, wasmFile string) ExampleContract {
 	anyAmount := sdk.NewCoins(sdk.NewInt64Coin("denom", 1000))
 	creator, _, creatorAddr := keyPubAddr()
 	fundAccounts(t, ctx, keepers.AccountKeeper, keepers.BankKeeper, creatorAddr, anyAmount)
 
-	wasmCode, err := ioutil.ReadFile("./testdata/hackatom.wasm")
+	wasmCode, err := ioutil.ReadFile(wasmFile)
 	require.NoError(t, err)
 
 	codeID, err := keepers.WasmKeeper.Create(ctx, creatorAddr, wasmCode, "", "", nil)
 	require.NoError(t, err)
-	return HackatomExampleContract{anyAmount, creator, creatorAddr, codeID}
+	return ExampleContract{anyAmount, creator, creatorAddr, codeID}
 }
 
 type HackatomExampleInstance struct {
```

Comment:
```suggestion
type ExampleContractInstance struct {
```

Reply:
This is not "any" contract instance but the Hackatom example. Some tests rely on this specific setup so that I do not want to mix this.

Reply:
I see, right. I did not read the code well enough.